examples/ACI Unified Playground/memory_agent.py

```
@@ -0,0 +1,413 @@
+                rprint("[green]WebSocket response sent.[/green]")
+
+    except WebSocketDisconnect:
+        rprint("[bold yellow]WebSocket connection closed.[/bold yellow]")
+    except Exception as e:
+        rprint(f"[bold red]An error occurred in WebSocket: {e}[/bold red]")
+        try:
+            await websocket.send_json(
+                {"type": "error", "sender": "System", "content": str(e)}
+            )
+        except:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)